src/EditorFeatures/Core/InlineRename/InlineRenameSession.cs

```diff
@@ -122,6 +124,9 @@ private set
 
     private readonly IInlineRenameInfo _renameInfo;
 
+    internal IInlineRenameInfo RenameInfo => _renameInfo;
+    internal JoinableTask<IInlineRenameLocationSet> AllRenameLocationsTask => _allRenameLocationsTask;
```

Comment:
slightly concerning we're exposing this.

Reply:
why do you need to expose this?

Reply:
They are needed in the Context building. 
e.g. to get the rename references/definition locations.

🤔This might look better if we refactoring the code structure, but I am fine with this approach now.